docs/source/conf.py

```diff
@@ -74,7 +74,13 @@
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ["_static"]
-
+html_context = {
+    "display_github": True, # Integrate GitHub
```

Comment:
Should I expect to see this link now in https://photonvision-docs--1862.org.readthedocs.build/en/1862/docs/quick-start/arducam-cameras.html ? I don't yet.